pages/components/Logo/getDimensions.ts

```diff
@@ -0,0 +1,62 @@
+type dimension = string | undefined;
+type dimensions = [string, string];
+
+export const aspectRatio = 926 / 1065;
+
+function validateMeasurement(str: string | undefined): Boolean {
+  const measurementRegex = /^\d+\.?\d*[a-zA-Z%]+$/;
+
+  if (str === undefined) {
+    return false;
+  }
+
+  if (!measurementRegex.test(str)) {
+    throw new Error(`
+      Value: ${str} is not a valid unit of measurement.
+        Please provide a string of number(s) followed by a unit of measurement (%, px, rem)
+    `);
+  }
+
+  return true;
+}
+
+function calculateMeasurement(measurement: string, calculate: Function): string {
```

Comment:
Function name `calculateMeasurement` is less than ideal - it can be confusing that it requires a measurement _and_ and `calculate` function. Without a typedef on `calculate`, it can be tough to infer what the function does with the measurement.